bundle/introducing_config/README.md

```diff
@@ -0,0 +1,158 @@
+      keys: [f_img, m_img]
+      image_only: True
+
+    - _target_: EnsureChannelFirstd
+      keys: [f_img, m_img]
+
+    - _target_: ScaleIntensityRanged
+      keys: [f_img, m_img]
+      a_min: 0.
+      a_max: 255.
+      b_min: 0.0
+      b_max: 1.0
+
+    - _target_: RandRotated
+      keys: [m_img]
+      range_x: $np.pi/4
+      prob: 1.0
+      mode: "bicubic"
+      keep_size: True
+
+    - _target_: RandZoomd
+      keys: [m_img]
+      min_zoom: 0.9
+      max_zoom: 1.1
+      prob: 1.0
+      mode: "bicubic"
+```
+
+`_target_` is a special key for the bundle parsing, it specifies the class to be instantiated. In this case, the `preprocessing` component is a `Compose` object. `Compose` is resolved to `monai.transforms.Compose` by default, so you don't need to specify the full path.
+
+`transforms` is the first argument to `monai.transforms.Compose`, it specifies the list of transforms to be added to the `Compose` object.
+
+The rest of the keys are the arguments to the transforms. In this case, the `LoadImaged` transform is used to load the images from the file paths, `EnsureChannelFirstd` transform is used to ensure the image data is in channel-first format, `ScaleIntensityRanged` transform is used to scale the image intensity to `[0, 1]`. `RandRotated` transform is used to randomly rotate the moving image, and `RandZoomd` transform is used to randomly zoom the moving image.
+
+## The equivalent Python code of the yaml config file
+
+```py
+import glob
+import matplotlib.pyplot as plt
+import monai
+import monai.transforms as mt
+import numpy as np
+
+url = "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/MedNIST.tar.gz"
+monai.apps.utils.download_and_extract(url, './mednist.tar.gz')
+
+dataset_dir = "MedNIST/Hand"
+datalist = list(sorted(glob.glob(f"{dataset_dir}/*.jpeg")))
+paired_dataset = [{"f_img": item, "m_img": item} for item in datalist]
+
+preprocessing = mt.Compose(
+    [
+        mt.LoadImaged(keys=("f_img", "m_img"), image_only=True),
+        mt.EnsureChannelFirstd(keys=("f_img", "m_img")),
+        mt.ScaleIntensityRanged(
+            keys=("f_img", "m_img"), a_min=0.0, a_max=255.0, b_min=0.0, b_max=1.0
+        ),
+        mt.RandRotated(
+            keys=["m_img"], range_x=np.pi / 4, prob=1.0, mode="bicubic", keep_size=True
+        ),
+        mt.RandZoomd(keys=["m_img"], min_zoom=0.9, max_zoom=1.1, prob=1.0, mode="bicubic"),
+    ]
+)
+
+first_pair = preprocessing(paired_dataset[0])
+print("displaying images:")
+plt.subplot(1, 2, 1)
+plt.imshow(first_pair["f_img"][0], cmap="gray")
+plt.subplot(1, 2, 2)
+plt.imshow(first_pair["m_img"][0], cmap="gray")
+plt.show()
+
+```
+
+## Topics not covered but possible in the config
+
+- Running customized Python components (made available on the `PYTHONPATH`).
+- Overriding the component in `example.yaml` using, for example, `--id=new_value` in the command line.
+- Multiple configuration files and cross-file references.
```

Comment:
Also for this one we can find an example in the model zoo, maybe we can show the link?
https://github.com/Project-MONAI/model-zoo/tree/dev/models/spleen_ct_segmentation